tests/lib/snaps/basic-iface-hooks-producer/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: basic-iface-hooks-producer
+version: 1.0
+slots:
+    bar:
+        interface: content
+        content: y
+        read:
```

Comment:
This is not allowed today. You can only expose one directory, not two. This is a long standing bug that I plan to fix as part of the work on fixing the content interface usability and bugs.